user.go

```diff
@@ -36,6 +57,11 @@ type User struct {
 
 	// Whether the user is a bot.
 	Bot bool `json:"bot"`
+
+	// The public flags on a users account.
```

Comment:
Super small nit, but if we’re already fixing something else: should be “user’s” :)